static/javascript/components/bar-charts/axis.js

```diff
@@ -1,9 +1,7 @@
 import React from "react";
 import * as d3 from "d3";
 
-export default class HorisontalBarChart2 extends React.Component {
-  static displayName = "Axis";
-
+export default class HorizontalBarChart2 extends React.Component {
```

Comment:
why did you remove `static displayName = "Axis";` ?

Reply:
I got an error when running it  15:10  error  'displayName' is not defined  no-undef
"eslint": "^3.15.0"

I bumped the version up to ^4.15.0, and it worked fine.

I did a search and it seems like it's a known issue with eslint:
https://github.com/eslint/eslint/issues/8720

I did a search for displayName in the files, and did not find anywhere that uses `displayName`, so I reckon it was safe to remove.

Reply:
displayName is used in React itself. https://reactjs.org/docs/react-component.html

Reply:
Ah, alright. Thanks for the doc. I could add them back.